### PR TITLE
Psd 488 apply version to pipeline

### PR DESCRIPTION
**Pull request by nickblows**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -18,36 +17,48 @@ def localSrcFolder = '.'
...
+      defraUtils.runTests(repoName, BUILD_NUMBER)
     }
     stage('Push container image') {
-      defraUtils.buildAndPushContainerImage(regCredsId, registry, imageName, containerTag)
-    }
+      defraUtils.buildAndPushContainerImage(regCredsId, registry, repoName, containerTag)
+    } 
+
     if (pr == '') {
+      stage('Verify version incremented') {
```
Comment (johnwatson484) [inline Jenkinsfile]: This needs to move to the else statement.  We want this to run on PR only so PR would not equal '' in that case.

File: Jenkinsfile [hunk 2]
```diff
@@ -84,17 +95,19 @@ node {
...
-        defraUtils.undeployChart(kubeCredsId, imageName, mergedPrNo)
+        defraUtils.undeployChart(kubeCredsId, repoName, mergedPrNo)
       }
     }
-    defraUtils.setGithubStatusSuccess()
+    stage('Set GitHub status as success'){
+      defraUtils.setGithubStatusSuccess()
+    }
   } catch(e) {
     defraUtils.setGithubStatusFailure(e.message)
```
Comment (johnwatson484) [inline Jenkinsfile]: Need to add Slack notification

Comment (nickblows) [reply Jenkinsfile]: added

File: Jenkinsfile (hunk 1 above)
Comment (nickblows) [reply Jenkinsfile]: Code moved inside else statement.

File: Jenkinsfile [hunk 3]
```diff
@@ -18,39 +17,51 @@ def localSrcFolder = '.'
...
 
-
 node {
   checkout scm
   try {
+    stage('Set GitHub status as pending'){
+      defraUtils.setGithubStatusPending()
+    }
     stage('Set branch, PR, and containerTag variables') {
       (pr, containerTag, mergedPrNo) = defraUtils.getVariables(repoName)
```
Comment (johnwatson484) [inline Jenkinsfile]: Need to reference the CSProj version number here using the shared library function

**Pull request merged**